### DISCUSSION TITLE
"noEnchantedGoldenAppleEating" >>> universal rule

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#69] [Ideas]
Being able to blacklist more items from eating than just the Enchanted Golden Apple.

**Comment by 1024-byteeeee** [comment]
Not considering expanding this rule. 😄 

If you want to prevent accidentally consuming a specific food, you can look for some client-side mods. This rule is mainly to prevent players from consuming the rare notch apple within the server.
